bin/docker/k8s/countly-ingress.yaml

```diff
@@ -7,37 +7,60 @@ data:
   tls.key: YOUR_SSL_KEY_IN_BASE64       # cat cert.key | base64
 type: kubernetes.io/tls
 ---
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: countly-ingress
-  annotations:
-    nginx.ingress.kubernetes.io/rewrite-target: /
+  namespace: countly
 spec:
+  ingressClassName: nginx
   tls:
   - hosts:
-    - YOUR_HOSTNAME                     # countly.example.com
+    - <required domain name>                     # example.count.ly
     secretName: countly-tls
   rules:
-  - http:
+  - host: <required domain name>                 # example.count.ly
```

Comment:
There is a lack of consistency for this requirement; the placeholder values I saw for the same parameter:
 - `<required domain name>`
 - `<required domain name>>`
 - `YOUR_HOST`

IMHO `YOUR_HOST` was okay. And why `example.count.ly` instead of `countly.example.com` as a comment? Second example is valid for non-Countly cloud customers.